packages/mobile/src/web3/dataEncryptionKey.ts

```diff
@@ -50,25 +50,9 @@ import {
   mtwAddressSelector,
 } from 'src/web3/selectors'
 import { estimateGas } from 'src/web3/utils'
-import { ec as EC } from 'elliptic'
-
-const ec = new EC('secp256k1')
-
 const TAG = 'web3/dataEncryptionKey'
 const PLACEHOLDER_DEK = '0x02c9cacca8c5c5ebb24dc6080a933f6d52a072136a069083438293d71da36049dc'
 
-// Based on https://github.com/celo-org/celo-monorepo/blob/f4dd249e9cef6a541b909c3dc7ae01888b088de2/packages/sdk/identity/src/odis/query.ts#L111
-export function signWithDEK({
-  message,
-  dataEncryptionKey,
-}: {
-  message: string
-  dataEncryptionKey: string
-}) {
-  const key = ec.keyFromPrivate(hexToBuffer(dataEncryptionKey))
```

Comment:
this was where we assumed the DEK provided was a private key (it was not)

Reply:
I tried to use "private" and "public" in all the variable names for DEK's to help prevent this kind of thing from happening again. It's very confusing to name both elements of a key pair with the same name and just rely on context clues to figure out which one it is. Also pretty dangerous..